frontend/pages/hosts/details/cards/Software/SelfService/SelfService.tsx

```diff
@@ -12,7 +12,11 @@ import { AxiosError } from "axios";
 
 import { NotificationContext } from "context/notification";
```

Comment:
Ability to click manual install status

@jacobshandling 's timeboxing to today,
@RachelElysia grab remainder of work